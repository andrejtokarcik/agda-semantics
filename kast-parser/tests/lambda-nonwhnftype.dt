expr = (Lambda x ((Lambda x (Set 1) x) (Set 0)) x)
