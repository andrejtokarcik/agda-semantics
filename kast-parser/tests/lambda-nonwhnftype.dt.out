<k>
    nf2 ( ( (Lambda x Set 0 x) , (Pi x Set 0 Set 0) ) )
</k>
<mgu>
    subst(.KList)
</mgu>
<tenv>
    .Map
</tenv>
<data>
    .Map
</data>
<meta>
    .Map
</meta>
